doc/logreg.txt

```diff
@@ -264,6 +264,20 @@ approximately 1.936 epochs/sec and it took 75 epochs to reach a test
 error of 7.489%. On the GPU the code does almost 10.0 epochs/sec. For this
 instance we used a batch size of 600.
 
+
+Prediction Using a Trained Model
++++++++++++++++++++++++++++++++
+
+``sgd_optimization_mnist`` serialize and pickle the model each time new
+lowest validation error is reached. We can reload this model and predict
+labels of new data. ``predict`` function shows an example of how
+this could be done.
+
+.. literalinclude:: ../code/logistic_sgd.py
+  :start-after:  ' ran for %.1fs' % ((end_time - start_time)))
```

Comment:
You can use `:pyobject: predict`, like explained in http://sphinx-doc.org/markup/code.html#includes:
>  If it is a Python module, you can select a class, function or method to include using the `pyobject` option

It is used in particular at https://github.com/memimo/DeepLearningTutorials/blob/predict/doc/logreg.txt#L219 in this file.
Otherwise, if it does not work, I would prefer to start and stop on explicit comments, as it is less likely to break later. For instance, if we add a new function after `predict`, then everything in that function until `if __name__ ...` wil get included as well, which is not what we want.